### PR TITLE
437 footer issue

### PR DESCRIPTION
**Pull request by mumptastic**
Resolves https://github.com/TAMULib/tamu-library-components/issues/437

Allows for the footer to always be pushed to the bottom of the page regardless of content.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "npm": ">=8.0.0"
   },
   "config": {
-    "DOCKER_SERVER": ""
+    "DOCKER_SERVER": "harbor.library.tamu.edu/weaver/wvr-components"
```
Comment (kaladay) [inline package.json:15]: I don't think we are supposed to have a default for this.
I imagine that you accidentally committed this.
Please remove.

**Review by kaladay — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (mumptastic) [reply package.json]: This should now be resolved @kaladay

**Review by kaladay — APPROVED** [review]

**Comment by jeremythuff** [comment]
We cannot style the body from within the component. This was part of the requirements for the initial implementation.

**Comment by wwelling** [comment]
https://github.com/TAMULib/weaver-components/pull/503

Two hours ago? Would of been preferred when work done and review requested.

**Pull request merged**